### PR TITLE
[containerapp] `az containerapp env create`: Update the default value of `--enable-workload-profiles` to `True`

### PR DESCRIPTION
**Pull request by Greedygre**
**Related command**

`az containerapp env create`: --enable-workload-profiles default value change to `True`
`az containerapp compose create`: fix containerapp invalid memory resource
Change the value of polling Timeout from 10min to 20min.

**Description**

az containerapp env create: Update the default value of --enable-workload-profiles to True.
Change the value of polling Timeout from 10min to 20min.

**Testing Guide**

**History Notes**

[Component Name 1] BREAKING CHANGE: `az command a`: Make some customer-facing breaking change
[Component Name 2] `az command b`: Add some customer-facing feature

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [ ] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
containerapp

File: src/azure-cli/azure/cli/command_modules/containerapp/tests/latest/test_containerapp_workload_profile_commands.py [hunk 1]
```diff
@@ -99,23 +99,25 @@ def test_containerapp_env_workload_profiles_e2e(self, resource_group):
...
             JMESPathCheck("name", env),
             JMESPathCheck("properties.provisioningState", "Succeeded"),
-            JMESPathCheck("properties.workloadProfiles", None)
+            JMESPathCheck("properties.workloadProfiles", None),
         ])
         self.cmd('containerapp env delete -g {} -n {} --yes --no-wait'.format(resource_group, env), expect_failure=False)
 
-        env1 = self.create_random_name(prefix='env', length=24)
-        self.cmd('containerapp env create -g {} -n {} --logs-destination none --enable-workload-profiles false'.format(
+        env1 = self.create_random_name(prefix='env4', length=24)
```
Comment (Juliehzl) [inline src/azure-cli/azure/cli/command_modules/containerapp/tests/latest/test_containerapp_workload_profile_commands.py:111]: env1

File: src/azure-cli/azure/cli/command_modules/containerapp/tests/latest/test_containerapp_workload_profile_commands.py [hunk 2]
```diff
@@ -99,23 +99,25 @@ def test_containerapp_env_workload_profiles_e2e(self, resource_group):
...
 
-        env = self.create_random_name(prefix='env4', length=24)
-        self.cmd('containerapp env create -g {} -n {} --logs-destination none'.format(
+        env = self.create_random_name(prefix='env', length=24)
+        self.cmd('containerapp env create -g {} -n {} --logs-destination none --enable-workload-profiles false'.format(
             resource_group, env), expect_failure=False, checks=[
             JMESPathCheck("name", env),
             JMESPathCheck("properties.provisioningState", "Succeeded"),
-            JMESPathCheck("properties.workloadProfiles", None)
+            JMESPathCheck("properties.workloadProfiles", None),
```
Comment (Juliehzl) [inline src/azure-cli/azure/cli/command_modules/containerapp/tests/latest/test_containerapp_workload_profile_commands.py:107]: , ?

File: src/azure-cli/azure/cli/command_modules/containerapp/tests/latest/test_containerapp_workload_profile_commands.py [hunk 3]
```diff
@@ -99,23 +99,25 @@ def test_containerapp_env_workload_profiles_e2e(self, resource_group):
...
             resource_group, env1), expect_failure=False, checks=[
             JMESPathCheck("name", env1),
             JMESPathCheck("properties.provisioningState", "Succeeded"),
-            JMESPathCheck("properties.workloadProfiles", None),
+            JMESPathCheck("length(properties.workloadProfiles)", 1),
+            JMESPathCheck('properties.workloadProfiles[0].name', "Consumption", case_sensitive=False),
+            JMESPathCheck('properties.workloadProfiles[0].workloadProfileType', "Consumption", case_sensitive=False),
         ])
-        self.cmd('containerapp env delete -g {} -n {} --yes --no-wait'.format(resource_group, env1), expect_failure=False)
+        self.cmd('containerapp env delete -g {} -n {} --yes --no-wait'.format(resource_group, env), expect_failure=False)
```
Comment (Juliehzl) [inline src/azure-cli/azure/cli/command_modules/containerapp/tests/latest/test_containerapp_workload_profile_commands.py:120]: delete env1?

**Review by Juliehzl — APPROVED** [review]

**Comment by yonzhan** [comment]
Please fix CI issues

**Comment by Greedygre** [comment]
Close this PR because merge main by mistake

**Pull request closed without merging**